vault/auth.go

```diff
@@ -460,10 +459,11 @@ func (c *Core) setupCredentials() error {
 		backend, err = c.newCredentialBackend(entry.Type, sysView, view, conf)
 		if err != nil {
 			c.logger.Error("core: failed to create credential entry", "path", entry.Path, "error", err)
-			if errwrap.Contains(err, ErrPluginNotFound.Error()) && entry.Type == "plugin" {
-				// If we encounter an error instantiating the backend due to it being missing from the catalog,
-				// skip backend initialization but register the entry to the mount table to preserve storage
-				// and path.
+			if entry.Type == "plugin" {
+				// If we encounter an error instantiating the backend due to an error,
+				// skip backend initialization but register the entry to the mount table
+				// to preserve storage and path.
+				c.logger.Warn("core: skipping plugin-based credential entry", "path", entry.Path)
```

Comment:
Can we output the error here so the user knows why it failed (plugin missing, bad SHA, etc.), and the same for mounts?

Reply:
The error is printed out up top in https://github.com/hashicorp/vault/pull/3686/files#diff-444ea30b36f7a95d6672fd6fd9b7a87cR461.

Reply:
Pfft, _obviously_ @bk, _**duh**_

Reply:
Ha, I even looked there and thought "Oh, it's only printing out the path", missed the error somehow :)